lib/iris/tests/benchmarking/regridding.py

```diff
@@ -0,0 +1,34 @@
+# Copyright Iris contributors
+#
+# This file is part of Iris and is released under the LGPL license.
+# See COPYING and COPYING.LESSER in the root of the repository for full
+# licensing details.
+
+"""
+This file currently exists as a simple demonstration of Airspeed Velocity
+performance testing. it is expected to be expanded/replaced in future.
+
+"""
+
+# import iris tests first so that some things can be initialised before
+# importing anything else.
+from iris import tests
+
+import iris
+
+from iris import analysis
+
+
+@tests.skip_data
+class RegriddingTests:
+    def setup(self):
+        # Prepare a cube and a regridding scheme.
+        file_path = tests.get_data_path(
+            ["NetCDF", "global", "xyt", "SMALL_hires_wind_u_for_ipcc4.nc"]
+        )
+        self.cube = iris.load(file_path)[0]
+        self.scheme_area_w = analysis.AreaWeighted()
+
+    def time_regrid_area_w(self):
+        # Regrid the cube onto itself.
+        self.cube.regrid(self.cube, self.scheme_area_w)
```

Comment:
@trexfeathers ... i.e., make this:
```python
self.cube.regrid(self.cube, AreaWeighted())
```

Reply:
https://github.com/SciTools/iris/pull/3526#discussion_r345729344